sashimi-webapp/src/database/sql-related/sqlCommands.js

```diff
@@ -0,0 +1,261 @@
+                                                       ' = ', folderId)])
+        .then(data => resolve(getArray(data)))
+        .catch(sqlError => reject(sqlError))
+      );
+    } else {
+      throw new exceptions.PromiseFunctionNotDefined();
+    }
+  };
+
+  this.loadFile = function loadFile(fileId) {
+    if (typeof Promise === 'function') {
+      return new Promise((resolve, reject) =>
+        alasql.promise([stringManipulator.stringConcat('SELECT ', constants.HEADER_FILE_MANAGER_FILE_MARKDOWN,
+                                                       ' FROM ', constants.ENTITIES_FILE_MANAGER,
+                                                       ' WHERE ', constants.HEADER_FILE_MANAGER_FILE_ID,
+                                                       ' = ', fileId)])
+        .then(data => resolve(getDataOutOfAlasql(data)))
+        .catch(sqlError => reject(sqlError))
+      );
+    } else {
+      throw new exceptions.PromiseFunctionNotDefined();
+    }
+  };
+
+  this.saveFile = function saveFile(fileId, markdownFile) {
+    if (typeof Promise === 'function') {
+      return new Promise((resolve, reject) => {
+        initPromiseSequence()
+        .then(() => alasql.promise([stringManipulator.stringConcat('UPDATE ', constants.ENTITIES_FILE_MANAGER,
+                                                                   ' SET ', constants.HEADER_FILE_MANAGER_FILE_MARKDOWN,
+                                                                   ' = "', markdownFile,
+                                                                   '" WHERE ', constants.HEADER_FILE_MANAGER_FILE_ID,
+                                                                   ' = ', fileId)])
+          .catch(sqlError => reject(sqlError)))
+        .then(() => {
+          const currentDateTime = dateTime.getCurrentDateTime();
+          alasql.promise([stringManipulator.stringConcat('UPDATE ', constants.ENTITIES_FILE_MANAGER,
+                                                         ' SET ', constants.HEADER_FILE_MANAGER_LAST_MODIFIED_DATE,
+                                                         ' = "', currentDateTime,
+                                                         '" WHERE ', constants.HEADER_FILE_MANAGER_FILE_ID,
+                                                         ' = ', fileId)])
+          .catch(sqlError => reject(sqlError));
+        })
+        .then(() => resolve(true))
+        .catch(sqlErr => reject(sqlErr));
+      });
+    } else {
+      throw new exceptions.PromiseFunctionNotDefined();
+    }
+  };
+
+  this.deleteFile = function deleteFile(fileId) {
+    if (typeof Promise === 'function') {
+      return new Promise((resolve, reject) =>
+        alasql.promise([stringManipulator.stringConcat('DELETE FROM ', constants.ENTITIES_FILE_MANAGER,
+                                                       ' WHERE ', constants.HEADER_FILE_MANAGER_FILE_ID,
+                                                       ' = ', fileId)])
```

Comment:
The SQL statements in this file could be vulnerable to SQL injections as parameters like `fileID`  and `folderID` are not sanitized.

One way could be to sanitize the parameters before using it in SQL statements.
Another alternative is to use [Escaping Query Values](https://github.com/mysqljs/mysql#escaping-query-values)